crates/re_types/definitions/rerun/components/keypoint_id.fbs

```diff
@@ -10,14 +10,14 @@ namespace rerun.components;
 
 /// A 16-bit ID representing a type of semantic keypoint within a class.
 ///
-/// \py `KeypointId`s are only meaningful within the context of a [`rerun.components.ClassDescription`][].
+/// \py `KeypointId`s are only meaningful within the context of a [`rerun.datatypes.ClassDescription`].
```

Comment:
aren't those now under the `rerun` namespace directly?

Reply:
We do import some things to the main `rerun` namespace, but I think for doclinks it is better to be explicit like htis